### DISCUSSION TITLE
A way for S3 assets to be appear as served from local site and not amazon s3?

### DISCUSSION THREAD

**Discussion by andreimoment** [craftcms/aws-s3#136] [Q&A]
Using the Pixel & Tonic S3 plugin, is there a way for assets to be served from `site.com/assetfolder`? 

Currently they are served from `s3.us-west-1.amazonaws.com/s3-bucket-name/assetfolder` which eliminates the possibility to associate SEO rankings for hosted PDFs with `site.com`.

Is there a way to have S3 assets appear to be served via `site.com/assetfolder` public URL instead?

Is there a setting in the S3 plug-in which allows for this?

**Comment by andris-sevcenko** [comment]
You'll want to set up a CNAME DNS record. Here's a[ relevant SE post](https://stackoverflow.com/a/34316651).

**Comment by andreimoment** [comment]
Thank you @andris-sevcenko.

How do you recommend I go about renaming the current bucket? Or maybe it would be better if I create a new one and move the assets there? Would this update the links in craft?

**Reply by andris-sevcenko** [reply]
You can probably rename it in the AWS console and then update the settings for the existing Volume in Craft CMs.
